scripts/nf1_classifier.py

```diff
@@ -117,178 +120,236 @@ def train_test_error(clf, train_x, train_y, test_x, test_y):
     test_err = roc_auc_score(test_y, pred_y, average='weighted')
     return train_err, test_err
 
-####################################
-# Perform Analysis
-####################################
+
+def get_cohens_d(score_df):
+    """
+    Find the effect size of decision functions for the input scoring dataframe
+
+    Arguments:
+    :param score_df: pandas DataFrame with decision function scores and status
+
+    Output:
+    Returns the Cohen's d effect size estimate statistic for two status groups
+    """
+    pos_samples = np.array(score_df[score_df['status'] == 1].decision.tolist())
+    neg_samples = np.array(score_df[score_df['status'] == 0].decision.tolist())
+    var_pos = np.sum((pos_samples - np.mean(pos_samples)) ** 2)
```

Comment:
Noting http://nbviewer.jupyter.org/github/AllenDowney/CompStats/blob/master/effect_size.ipynb -- see cohen's d cell if you're to verify implementations.